handbook/engineering/distribution/goals_completed.md

```diff
@@ -0,0 +1,118 @@
+# Distribution team completed goals
+
+This page records goals completed by the Distribution team historically. See also our [current goals](goals.md).
+
+- **2020**
+  - [FY-20-Q4](#FY-20-Q1)
+  - [FY-20-Q3](#FY-20-Q1)
+  - [FY-20-Q2](#FY-20-Q1)
+  - [FY-20-Q1](#FY-20-Q1)
+- **2019**
+  - [FY-19-Q4](#FY-19-Q4)
+
+## FY-20-Q4
+
+### Automatic e2e testing
+
+[RFC 137](https://docs.google.com/document/d/14f7lwfToeT6t_vxnGsCuXqf3QcB5GRZ2Zoy6kYqBAIQ/edit#heading=h.trqab8y0kufp) saw us remove our e2e tests from CI entirely because it was unreliable. Now e2e tests are ran as part of our monthly release process completely manually, and are heavily broken/outdated each time we attempt to do it. Fixing them often takes ~1.5d of work from a developer on the team. Per the RFC, we want to run these e2e tests on CI in an automated and reliable fashion.
+
+- [Tracking issue](https://github.com/sourcegraph/sourcegraph/issues/10646)
+- Discussions: [RFC 137](https://docs.google.com/document/d/14f7lwfToeT6t_vxnGsCuXqf3QcB5GRZ2Zoy6kYqBAIQ/edit#heading=h.trqab8y0kufp)
+- Owner: Dave
+- Dependencies:
+  - [CI infrastructure that can run Docker containers in a reliable way](#ci-infrastructure-that-can-run-docker-containers-in-a-reliable-way)
+
+### CI infrastructure that can run Docker containers in a reliable way
+
+Today we cannot release of Sourcegraph, run e2e tests, or perform Docker image tests in an automated fashion because our CI infrastructure does not support running Docker containers (or VMs/Vagrant) in a reliable way. Today we have a [side-car DIND container in our CI pipeline](https://sourcegraph.sgdev.org/search?q=repo:%5Esourcegraph/infrastructure%24+dind&patternType=literal) but it is flaky, unreliable, and a regular source of issues which has led to us removing automated testing (see [Automatic e2e testing](#automatic-e2e-testing)).
```

Comment:
```suggestion
Today we cannot release of Sourcegraph, run e2e tests, or perform Docker image tests in an automated fashion because our CI infrastructure does not support running Docker containers (or VMs/Vagrant) in a reliable way. We have a [side-car DIND container in our CI pipeline](https://sourcegraph.sgdev.org/search?q=repo:%5Esourcegraph/infrastructure%24+dind&patternType=literal) but it is flaky, unreliable, and a regular source of issues which has led to us removing automated testing (see [Automatic e2e testing](#automatic-e2e-testing)).
```